cirq-core/cirq/value/condition.py

```diff
@@ -87,8 +99,8 @@ class SympyCondition(Condition):
     """A classical control condition based on a sympy expression.
 
     This condition resolves to True iff the sympy expression resolves to a
-    Truthy value when the measurement keys are substituted in as the free
-    variables.
+    truthy value (i.e. `bool(x) == True`) when the measurement keys are
+    substituted in as the free variables.
 
     To account for the fact that measurement key strings can contain characters
     not allowed in sympy variables, we use x0..xN for the free variables and
```

Comment:
I get the sense that an explicit dict from "expr-keys" to control keys would be cleaner here.

Reply:
`with_keys` goes in order. But perhaps `with_keys` should use a `{from:to}` format as well.